plugins/repoquery.py

```diff
@@ -97,12 +100,16 @@ def parse_arguments(args):
                         help=_('show only results there provides REQ'))
     parser.add_argument('--whatrequires', metavar='REQ',
                         help=_('show only results there requires REQ'))
+    parser.add_argument("--alldeps", action="store_true",
+                        help="shows results that requires package provides and files")
     parser.add_argument('--querytags', action='store_true',
                         help=_('show available tags to use with '
                                '--queryformat'))
     parser.add_argument('--resolve', action='store_true',
                         help=_('resolve capabilities to originating package(s)')
-                       )
+                        )
+    parser.add_argument("--tree", action="store_true", 
+                        help="For the given packages print a tree of the packages.")
```

Comment:
Hm, I think that the arguments should be reimplemented. It should be possible to use `--queryformat` at the same time as e.g. `--requires`. `--requires` should set some other option than `queryformat` (it should set `queryformat` as well but only if `--queryformat` is not used) and thus `--queryformat` should not be in the same mutually exclusive group. Then `--tree` could be in the same mutually exclusive group as e.g. `--info` and `tree_seed` would not have to search for `"%{requires}"` in the `queryformat`. But let's not complicate your task. We can change it later...